copyright.md

```diff
@@ -17,10 +15,8 @@ The manual is provided “as is” without expressed or implied warranty. You ma
 *	https://computation.llnl.gov/casc/sundials/main.html
```

Comment:
Please also change `(c)2002` to `(c)2015` for the solver (that's the version we're currently using; should upgrade to the latest one at the begin of 2019)